src/commands/txCommands/pushNewContent.js

```diff
@@ -36,13 +36,14 @@ const pushResource$ = poData =>
 		.flatMap(([gitBranch, branchResourceExists]) => {
 			if (Object.keys(poData).length) {
 				const pushed$ = branchResourceExists
-					? txlib.updateResource$
-					: txlib.createResource$;
+					? txlib.updateResource$.do(() => console.log(`new content - update ${gitBranch}`))
```

Comment:
would it be useful to also log `Object.keys(poData).length`? Would that be the number of new keys or something?

Reply:
Knowing the number of keys would be a useful check on the result of diffing new content against old. However, this is a situation where concise and useful information do not coexist. 

But I can add it if you think it would be helpful.